### ISSUE TITLE
Any plans to integrate the content management api as well?

### ISSUE THREAD

**Issue by PhilippSchreiber** [jvandemo/angular-contentful#4]
Hey @jvandemo,
thanks for the great lib. Do you have any plans to integrate to the contentful content management api as well?
Cheers
Philipp

**Comment by jvandemo** [comment]
Thanks @PhilippSchreiber, I didn't have plans to do so, but I'll have a look at the [original contentful management library](https://github.com/contentful/contentful-management.js) to see how much work it would take me.

Would you have an interest in using it? Thanks!

**Comment by PhilippSchreiber** [comment]
Thanks for your reply. Yes I'm interested in using it... As far as I have seen only the POST and PUT parts are missing since the content management api and the content delivery api use the same endpoints to get the existing contents... The oauth part is perhaps the hardest one ...

**Comment by BradSiegler** [comment]
@jvandemo I agree with @PhilippSchreiber for the most part. It would be great if an angular integration for the content management api provided the ability/tools to build a service that handles auth and creating/editing/deleting spaces (not retrieving, as this lib facilitates that perfectly). It would provide a ton of flexibility interfacing with Contentful through people's own apps.

**Comment by eyesurewould** [comment]
Partial support of the content management API to support create and update of entries would be a huge step forward. I recently completed a project where I incorporated both the content API and the content management API in a single service and I've been contemplating releasing that service as open source.

Of course, that was before I found this package. Now, I'll consider just contributing here versus releasing a competing package that does the same things. @jvandemo are you looking for contributors currently?
